community/import-tool/src/main/java/org/neo4j/tooling/ImportTool.java

```diff
@@ -181,7 +183,11 @@
                 "<true/false>",
                 "Whether or not to ignore extra columns in the data not specified by the header. "
                         + "Skipped columns will be logged, containing at most number of entities specified by "
-                        + BAD_TOLERANCE.key() + "." );
+                        + BAD_TOLERANCE.key() + "." ),
+        DATABASE_CONFIG( "db-config", null,
+                "<path/to/neo4j.properties>",
+                "(advanced) File specifying database-specific configuration. For more information consult "
+                        + "manual about available configuration options for a neo4j configuration file" );
```

Comment:
perhaps we should explicitly call out the block sizes?
We should probably also note that *only store-structure related* properties are used? e..g NOT dbms.pagecache.memory or keep_logical_logs or legacy-auto-index config.